pkg/k8sutil/k8sutil.go

```diff
@@ -383,3 +385,45 @@ func mergeMapsByPrefix(from map[string]string, to map[string]string, prefix stri
 
 	return to
 }
+
+// DiffRulerConfigMap preProvisionRuleConfigMapNumber can't less than newConfigMaps number.
```

Comment:
Comment seems to be outdated
```suggestion
// DiffRulerConfigMap preProvisionRuleConfigMapNumber can't less than newConfigMaps number.
```